### DISCUSSION TITLE
Excluding private IP ranges breaks private DNS

### DISCUSSION THREAD

**Discussion by eiqnepm** [strongswan/strongswan#864] [Q&A]
I'm trying to replicate the allow LAN feature most VPN apps have. When I exclude the private IP range `10.0.0.0/8` from being routed through the VPN tunnel Android's private DNS feature stops working. Does anyone have any idea why this is happening?

![](https://drive.google.com/uc?id=1-ghxRQn7eoiNqGgb1pIcqfqM4ZcOkLh7)
![](https://drive.google.com/uc?id=1a9_bmUm1GLvHaYRZ5mMe4bQzuQClU5HO)
![](https://drive.google.com/uc?id=1Yi8UpoV38Utql2ljrIorlbSAAXS2Ly0l)
![](https://drive.google.com/uc?id=1Pl6ws1dLhwp4woLVDXKS7tpsjngeuipQ)

**Comment by tobiasbrunner** [comment]
> When I exclude the private IP range `10.0.0.0/8` from being routed through the VPN tunnel Android's private DNS feature stops working.

Is it that range specifically? Or doesn't the range matter?

*Edit:* OK, saw the other screenshot, somehow didn't load initially. No idea what's going on there, you might want to ask Google about it.
